content/applications/finance/fiscal_localizations/argentina.rst

```diff
@@ -948,6 +948,87 @@ the corresponding layout and fiscal stamps stated in the :ref:`Invoice printed r
+remove any active filters. Then, find the documents and activate them by toggling the option
+:guilabel:`Active`.
+
+Purchase journal
+****************
+
+A purchase journal is needed to generate an electronic vendor bill with a document type *Liquidity
+Product*. This journal needs to have an active synchronization with the AFIP as this electronic
+document will be sent to the AFIP.
+
+To modify the existing purchase journal, or create a new purchase journal, navigate to
+:menuselection:`Accounting --> Configuration --> Journals`. Then, click the :guilabel:`New` button,
+and fill in the following required information:
+
+- :guilabel:`Type`: select :guilabel:`Purchase` from the drop-down menu for vendor bill journals.
+- :guilabel:`Use Documents`: check this field for the journal to select the electronic document type
+  (in this case *sales account* and *liquid product*).
+- :guilabel:`Is AFIP POS`: select this field for the journal to be able to generate electronic
+  documents (in this case *sales account* and *liquid product*).
+- :guilabel:`AFIP POS System`: select :guilabel:`Electronic Invoice - Web Service` from the
+  drop-down menu in order to send the electronic document to AFIP via web service.
+- :guilabel:`AFIP POS Number`: is the number configured in the AFIP to identify the operations
+  related to this AFIP POS.
+- :guilabel:`AFIP POS Address`: is the field related to the commercial address registered for the
+  POS, which is usually the same address as the company. For example, if a company has multiple
+  stores (fiscal locations) then the AFIP will require the company to have one AFIP POS per
+  location. This location will be printed in the invoice report.
+
+.. image:: argentina/l10n-ar-purchase-journal.png
+   :align: center
+   :alt: l10n_ar Purchase Journal Configuration.
+
+Sales journal
+*************
+
+A sales journal is needed to register the invoice when a product is sold to a third party that will
+then sell the same product. This journal will not be synced with AFIP as the invoice will not be
+electronic.
+
+To modify the existing purchase journal or create a new purchase journal, navigate to
+:menuselection:`Accounting --> Configuration --> Journals`. Then, click the :guilabel:`New` button,
+and fill in the following required information:
+
+- :guilabel:`Type`: select :guilabel:`Sales` from the drop-down menu for sale invoice journals.
+- :guilabel:`Use Documents`: check this field on the journal to select the electronic document type
+  (in this case the electronic invoice).
+
+.. image:: argentina/l10n-ar-sales-journal.png
+   :align: center
+   :alt: l10n_ar Sales Journal Configuration.
+
+Invoicing flow
+~~~~~~~~~~~~~~
+
+Once the configurations are all set, the *Liquidity Product Vendor Bill* will be generated by the
+company that is selling the product on behalf of another party. For example, a distributor of a
+specific product.
```

Comment:
That seems clearer than what is explained in the intro. Maybe better introduce the feature?